lib/miq_automation_engine/engine/miq_ae_engine.rb

```diff
@@ -261,6 +261,15 @@ def self.create_automation_attribute_array_value(value)
     end.join(",")
   end
 
+  def self.create_automation_attributes_from_obj_array(objects, attrs_hash)
```

Comment:
It seems inconsistent, to me, for this method to be named `create_`, when it takes `attrs_hash` as an argument (== doesn't create anything), and doesn't return the mutated hash.

`obj_array` is also a little awkward to read... would `from_array` or `from_objects` work?